DependencyInjection/Compiler/DoctrineMongoDBMappingsPass.php

```diff
@@ -28,8 +28,8 @@ class DoctrineMongoDBMappingsPass extends RegisterMappingsPass
      * @param Definition|Reference $driver            the driver to use
      * @param array                $namespaces        list of namespaces this driver should handle
      * @param string[]             $managerParameters list of parameters that could tell the manager name to use
-     * @param bool                 $enabledParameter  if specified, the compiler pass only
-     *                                                executes if this parameter exists in the service container.
+     * @param string|false         $enabledParameter  if specified, the compiler pass only
```

Comment:
I changed these in https://github.com/doctrine/DoctrineMongoDBBundle/commit/3879b97bb3c0a61b66cef5dd1cf7972a5ca7e2f0, but didn't check with the parent one:

https://github.com/symfony/symfony/blob/61ecf9bf6a934c5df680008daa7b6642742f8bf2/src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterMappingsPass.php#L67-L74

now should be fine.